CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -95,25 +95,53 @@ export default function Report({ reportID }) {
     },
     {
       header: "Duplicates",
-      accessorFn: (row) => row.TotalDuplicates.toString(),
+      accessorFn: (row) => row.TotalDuplicates,
```

Comment:
accessorFn needs to have the .toString() method at the end for filtering to work properly. Make sure to change this back for all the columns where changed.